st/initscripts

```diff
@@ -27,12 +28,61 @@ set -e
 M0_SRC_DIR=`readlink -f $0`
 M0_SRC_DIR=${M0_SRC_DIR%/*/*}
 
-. $M0_SRC_DIR/utils/functions # report_and_exit
+
+m0t1fs_dir="$M0_SRC_DIR/m0t1fs/linux_kernel/st"
+. $m0t1fs_dir/common.sh
+. $m0t1fs_dir/m0t1fs_common_inc.sh
+. $m0t1fs_dir/m0t1fs_client_inc.sh
+. $m0t1fs_dir/m0t1fs_server_inc.sh
 
 cd $M0_SRC_DIR
 
-sudo ./scripts/install-motr-service -u
-sudo ./scripts/install-motr-service -l
-sudo /usr/bin/time -v ./st/m0t1fs -vv
+
+N=4
+K=2
+S=0
+P=12
+stride=32
+
+motr_service_start()
+{
+	local n=$1
+	local k=$2
+	local s=$3
+	local p=$4
+	local stride=$5
+	local multiple_pools=0
+
+	echo "n k s p:$N $K $S $P"
+	motr_service start $multiple_pools $stride $n $k $s $p
+	if [ $? -ne 0 ]
+	then
+		echo "Failed to start Motr Service..."
+		return 1
+	fi
+	echo "motr service started"
+
+	ios_eps=""
+	for ((i=0; i < ${#IOSEP[*]}; i++)) ; do
+		ios_eps="$ios_eps -S ${lnet_nid}:${IOSEP[$i]}"
+	done
+	return 0
+}
+
+motr_service_stop()
+{
+	motr_service stop
+	if [ $? -ne 0 ]
+	then
+		echo "Failed to stop Motr Service..."
+		return 1
+	fi
+}
+
+motr_service_start $N $K $S $P $stride 
+echo "CORTX- Motr Service is started"
+motr_service_stop
+echo "CORTX- Motr Service is stopped"
+
 
 report_and_exit initscripts $?
```

Comment:
Here, the $? is the return value of last comment, that is `echo`.
But we need to catch the result of `motr_service_start()`.
So we need something like this:
```
motr_service_start $N $K $S $P $stride 
rc=$?
echo "CORTX- Motr Service is started"
motr_service_stop
echo "CORTX- Motr Service is stopped"


report_and_exit initscripts $rc
```

Reply:
Done the changes, and ran the ST in a loop for 50 times and it ran fine. Also attaching the log file below of the same.
[st_op.log](https://github.com/Seagate/cortx-motr/files/9474982/st_op.log)